Fitting Inspector Issues view into docs (#846)
avohq/docs: pages/workspace/inspector/inspector-slack-alerts.mdx

---
layout: 'guide'
title: 'Inspector Slack Alerts'
abstract: 'How to start recieving Slack alerts for your Inspector issues'
---

import Link from '../../../components/Link';

> On Inspector tab you can connect Inspector to your Slack to receive alerts when new issues appear in your production environments.

Reviewer: ```suggestion
In the Inspector events view you can connect Inspector to your Slack to receive alerts when new issues appear in your production environments.
```
